violet-framework/src/main/java/com/horstmann/violet/workspace/editorpart/behavior/AbstractEditorPartBehavior.java

```diff
@@ -145,55 +145,62 @@ public void onToolSelected(GraphTool selectedTool)
         // TODO Auto-generated method stub
```

Comment:
Jeżeli znajdę pull request który posiada ToDo to rozumiem że wszystko cofniesz tak?